File: verificare-numar-palindrom/fisier.cpp
#include <iostream>
using namespace std;
bool verifpal(int a) {
    int copie = a, oglinda = 0;
    while (a > 0) {
        oglinda = oglinda * 10 + a % 10;
        a = a / 10;
    } if (copie == oglinda)
        return true;
    return false;
}
int main() {
    int a;
    cin >> a;
    if (verifpal(a))
        cout << "nr este palindrom";
    else
        cout << "nr nu este palindrom";
    return 0;
}

File: verificare-numar-palindrom/fisier.java
import java.util.Scanner;
class Tutorial {
    public static boolean verifpal(int a) {
        int copie = a, oglinda = 0;
        while (a > 0) {
            oglinda = oglinda * 10 + a % 10;
            a = a / 10;
        } if (oglinda == copie)
            return true;
        return false;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        if (verifpal(a))
            System.out.println("nr este palindrom");
        else
            System.out.println("nr nu este palindrom");
    }
}
